<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDataSourcePage</class>
 <widget class="QWidget" name="CreateDataSourcePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateDataSourcePage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stacked_widget">
     <widget class="CreateStandardDataSource" name="create_standard_data_source"/>
     <widget class="CreateFileDataSource" name="create_file_data_source"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CreateStandardDataSource</class>
   <extends>QWidget</extends>
   <header>..create.createStandardDataSource</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreateFileDataSource</class>
   <extends>QWidget</extends>
   <header>..create.createFileDataSource</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
